lib/Doctrine/DBAL/Driver/DrizzlePDOMySql/Driver.php

```diff
@@ -34,15 +34,15 @@ public function createDatabasePlatformForVersion($version)
     /**
      * {@inheritdoc}
      */
-    public function getDatabasePlatform()
+    public function getDatabasePlatform() : DrizzlePlatform
```

Comment:
Isn't it a breaking change?

Reply:
Woops it is indeed, since the class isn't final… I will use `@return` phpdoc instead.